core/src/main/java/org/vertexium/query/Query.java

```diff
@@ -63,6 +63,22 @@
     <T> Query range(String propertyName, T startValue, boolean inclusiveStartValue, T endValue, boolean inclusiveEndValue);
 
     /**
+     * Adds an id filter to the query.
+     *
+     * @param ids The ids to filter on.
+     * @return The query object, allowing you to chain methods.
+     */
+    Query hasId(String... ids);
+
+    /**
+     * Adds an id filter to the query.
+     *
+     * @param ids The ids to filter on.
+     * @return The query object, allowing you to chain methods.
+     */
+    Query hasId(Collection<String> ids);
```

Comment:
Can you make this `Iterable` instead of `Collection`

Reply:
Fixed with https://github.com/visallo/vertexium/pull/175/commits/095e4883c02e595c40723a876504f7da2fd80ced